airflow/models/__init__.py

```diff
@@ -28,6 +28,7 @@
 from airflow.models.pool import Pool
 from airflow.models.renderedtifields import RenderedTaskInstanceFields
 from airflow.models.sensorinstance import SensorInstance  # noqa: F401
+from airflow.models.serialized_dag import SerializedDagModel
```

Comment:
This appears to be here to be able to do `models.SerializedDagModel` in www.views, right?

If so could you change _that_ to use ` from airflow.models.serialized_dag import SerializedDagModel`?

We want to move away from mass-auto-importing things in Airflow (as it's a slowdown for initial process startup, and makes dependency cycles more likely).